ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/StepSchemaResolver.java

```diff
@@ -293,6 +295,13 @@ private LogicalSchema handleTableSelect(
     return buildSelectSchema(schema, step.getKeyColumnNames(), step.getSelectExpressions());
   }
 
+  private LogicalSchema handleTableSuppress(
+      final LogicalSchema schema,
+      final TableSuppress<?> step
+  ) {
+    return buildSuppressSchema(schema);
```

Comment:
any reason to have another method to call out to instead of inlining `return schema` here?

Reply:
No we can simply just use the existing method `sameSchema`  instead of creating `handleTableSuppress` and `buildSuppressSchema` methods, since Suppress does not alter the schema